File: android/src/main/kotlin/com/mytiki/sdk/capture/receipt/capacitor/req/ReqLogin.kt
/*
 * Copyright (c) TIKI Inc.
 * MIT license. See LICENSE file in root directory.
 */

package com.mytiki.sdk.capture.receipt.capacitor.req

import com.getcapacitor.JSObject

class ReqLogin(data: JSObject) {
    val username: String?
    val password: String?
    val provider: String?

    init {
        username = data.getString("username")
        password = data.getString("password")
        provider = data.getString("provider")
    }
}

File: ios/Plugin/Src/Req/ReqLogin.swift
/*
 * Copyright (c) TIKI Inc.
 * MIT license. See LICENSE file in root directory.
 */

import Foundation
import Capacitor

public class ReqLogin {
    var username: String
    var password: String
    var source: String
    
    init (data: CAPPluginCall){
        username = data.getString("username") ?? ""
        password = data.getString("password") ?? ""
        source = data.getString("source") ?? ""
    }
}
